quirks/websites-with-shared-credential-backends.json

```diff
@@ -197,5 +197,14 @@
     [
         "lookmark.io",
         "lookmark.link"
+    ],
+    [
+        "claro.com.br",
+        "auth.netcombo.com.br",
+        "net.com.br"
+    ],
+    [
+        "vivo.com.br",
+        "telefonica.com.br"
     ]
```

Comment:
There is a login page at each of these websites that lets you log in with the same credentials?

Reply:
Here I will remove telefonica.com.br